### ISSUE TITLE
install.sh (0.11.0) fails to find the the packageserver csv

### ISSUE THREAD

**Issue by gearoidibm** [operator-framework/operator-lifecycle-manager#981]
The following lines of install.sh fail to find the packageserver csv although the csv is actually present.
To recreate:
Run `install.sh 0.11.0`
On the cluster the package server is present:
```
$ kubectl get csv -n olm
NAME            DISPLAY          VERSION   REPLACES   PHASE
packageserver   Package Server   0.11.0               Succeeded
 
```
Run the following snippet from install.sh
```
retries=50
until [[ $retries == 0 || $new_csv_phase == "Succeeded" ]]; do
    new_csv_phase=$(kubectl get csv -n "${namespace}" packageserver.v${release} -o jsonpath='{.status.phase}' 2>/dev/null || echo "Waiting for CSV to appear")
    if [[ $new_csv_phase != "$csv_phase" ]]; then
        csv_phase=$new_csv_phase
        echo "Package server phase: $csv_phase"
    fi
    sleep 1
    retries=$((retries - 1))
done

if [ $retries == 0 ]; then
    echo "CSV \"packageserver\" failed to reach phase succeeded"
    exit 1
fi
```

The output is:
```
Package server phase: Waiting for CSV to appear
CSV "packageserver" failed to reach phase succeeded
```

Environment:
```
$ oc version
oc v3.11.117
kubernetes v1.11.0+d4cacc0
features: Basic-Auth

Server
openshift v3.11.117
kubernetes v1.11.0+d4cacc0
```

**Comment by WtfJoke** [comment]
I experience the same problem (with` install.sh` and also locally with `make run-local` on minikube)

The command:
`kubectl rollout status -w deployment/packageserver --namespace=olm`
outputs:
`deployment "packageserver" successfully rolled out`

**Comment by a-hat** [comment]
I have the same issue. It seems that in this line of the install script, the name of the packageserver is specified incorrectly:

`new_csv_phase=$(kubectl get csv -n "${namespace}" packageserver.v${release} -o jsonpath='{.status.phase}' 2>/dev/null || echo "Waiting for CSV to appear")`

When I execute `kubectl get csv -n olm packageserver.v0.11.0`, nothing is found. The correct name seems to be just `packageserver`, without the version suffix.

In the `install_local.sh` script, the line is different (without the version suffix):

`new_csv_phase=$(kubectl get csv -n "${namespace}" packageserver -o jsonpath='{.status.phase}' 2>/dev/null || echo "Waiting for CSV to appear")`

**Comment by ecordell** [comment]
Thanks for this report! Fixed the script on the 0.11.0 release.
